src/runtime/runtimeClient.ts

```diff
@@ -81,9 +81,21 @@ export abstract class RuntimeClient {
     }
 
     this.evaluation = ev;
+
+    this.updateStyles();
+
     return ev;
   }
 
+  updateStyles() {
+    var css = "";
+    this.lastParse.code.replace(/```css\n([\w\W]*?)```/g, (g0, g1) => { // \n excludes disabled blocks
+      css += g1;
+    });
+
+    document.getElementById("app-styles").innerHTML = css;
```

Comment:
The `runtimeClient` is sort of unfortunately-named. It is the client for the runtime, but it lives wherever the the runtime is executing, which may be the server. Because of this, you can't reliably access `document` here. You'll need to send the parsed CSS to the actual client as part of the parse message and have the client update the styles.